app/src/main/java/com/ciandt/dragonfly/example/shared/BaseActivity.kt

```diff
@@ -72,7 +72,7 @@ abstract class BaseActivity : AppCompatActivity(), DebugActionsHelper.Debuggable
         return debugDrawer
     }
 
-    fun configDebugDrawer(currentIntent: Intent) {
+    open fun configDebugDrawer(currentIntent: Intent) {
```

Comment:
Instead of overriding the debug drawer, wouldn't it be easier to add an instance variable (hasDebugDrawer; default to true), which children set?